### PR TITLE
Opt viz

### PR DESCRIPTION
**Pull request by anitagraser**
The goal of this PR is to make plotting dependencies optional as far as possible: https://github.com/anitagraser/movingpandas/issues/78

### CODE REVIEW

**Comment by raybellwaves** [comment]
I we can add testing for the new env similar to https://github.com/pydata/xarray/blob/main/.github/workflows/ci.yaml#L47

**Comment by raybellwaves** [comment]
> I we can add testing for the new env similar to https://github.com/pydata/xarray/blob/main/.github/workflows/ci.yaml#L47

Added this to https://github.com/anitagraser/movingpandas/pull/261

**Comment by anitagraser** [comment]
Let's see if we can use geopandas-base instead of geopandas, as [suggested by Joris](https://github.com/conda-forge/movingpandas-feedstock/pull/16#discussion_r1035657790)

**Comment by raybellwaves** [comment]
> Let's see if we can use geopandas-base instead of geopandas, as [suggested by Joris](https://github.com/conda-forge/movingpandas-feedstock/pull/16#discussion_r1035657790)

Tried locally and didn't work. Probably worth making a separate PR

**Comment by anitagraser** [comment]
Mhm, readthedocs does not like matplotlib-base

**Comment by raybellwaves** [comment]
> Mhm, readthedocs does not like matplotlib-base

I created https://github.com/anitagraser/movingpandas/issues/262 to fix the rtd issue

**Comment by raybellwaves** [comment]
when I tried with geopandas-base locally I got things like

```
___________________ ERROR collecting movingpandas/tests/test_overlay.py ____________________
ImportError while importing test module '/Users/ray/Documents/PYTHON_dev/movingpandas/movingpandas/tests/test_overlay.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
../../../miniforge3/envs/mpd-mini/lib/python3.10/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
movingpandas/tests/test_overlay.py:7: in <module>
    from movingpandas.tests.test_trajectory import Node, make_traj, CRS_METRIC, CRS_LATLON
movingpandas/tests/test_trajectory.py:9: in <module>
    from fiona.crs import from_epsg
E   ModuleNotFoundError: No module named 'fiona'
```

File: environment-minimum.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+name: mpd-mini
+channels:
+  - conda-forge
+dependencies:
+  - python=3.10
+  - geopandas-base
+  - fiona
```
Comment (raybellwaves) [inline environment-minimum.yml:7]: ah I see fiona has been added in here

**Review by raybellwaves — APPROVED** [review]

File: environment-minimum.yml (hunk 1 above)
Comment (anitagraser) [reply environment-minimum.yml:7]: Yes, fiona and rtree had to be added explicitly

**Pull request merged**